src/Directory.Build.props

```diff
@@ -2,79 +2,18 @@
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(OutDirName)' == ''">
-    <OutDirName Condition="$(IsUnitTestProject)">UnitTests</OutDirName>
-    <OutDirName Condition="$(IsIntegrationTestProject)">IntegrationTests</OutDirName>
     <OutDirName Condition="'$(OutDirName)' == ''">Dlls</OutDirName>
   </PropertyGroup>
 
   <Import Project="..\Directory.Build.props"/>
-  <Import Project="$(RepoToolsetDir)Settings.props" Condition="'$(ExcludeRestorePackageImports)' != 'true' AND Exists('$(RepoToolsetDir)Settings.props')" />
   <Import Project="..\build\import\VisualStudio.XamlRules.props" />
 
   <PropertyGroup>
-    <_IsVisualStudioDeveloperBuild Condition="'$(OfficialBuild)' != 'true' AND '$(CIBuild)' != 'true' AND '$(BuildingInsideVisualStudio)' == 'true'">true</_IsVisualStudioDeveloperBuild>
-
-    <!-- Code Coverage doesn't currently work for portable/embedded PDBs - force to full inside Visual Studio builds -->
-    <DebugType Condition="'$(_IsVisualStudioDeveloperBuild)' == 'true'">full</DebugType>
-  </PropertyGroup>
-  
-  <PropertyGroup>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Language)' == 'C#'">
-    <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
-    <LangVersion>8.0</LangVersion>
-    <!-- New name -->
-    <Nullable>enable</Nullable>
-    
-    <!-- Old name while we wait for new name to be everywhere -->
-    <NullableContextOptions>enable</NullableContextOptions>
-    <Features>strict</Features>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Language)' == 'VB'">
-    <!-- Build machines only have Visual Studio 15.5 (which comes with VB 15.3) -->
-    <LangVersion>15.3</LangVersion>
-    <!-- The default is Embed -->
-    <VBRuntime>Default</VBRuntime>
-    <RemoveIntegerChecks>true</RemoveIntegerChecks>
-    <Features>strict</Features>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <CodeAnalysisRuleSet Condition="'$(IsTestProject)' == 'true'">$(RepoRoot)build\ruleset\Test.ruleset</CodeAnalysisRuleSet>
-    <CodeAnalysisRuleSet Condition="'$(IsTestProject)' != 'true'">$(RepoRoot)build\ruleset\Default.ruleset</CodeAnalysisRuleSet>
-
-    <TestArchitectures>x86</TestArchitectures>
-
-    <!-- Use IBC optimization data if available -->
-    <IbcOptimizationDataDir Condition="$(EnableIbc) == 'true'">$(NuGetPackageRoot)\RoslynDependencies.ProjectSystem.OptimizationData\$(RoslynDependenciesProjectSystemOptimizationDataVersion)\content\OptimizationData\</IbcOptimizationDataDir>
-
-    <UseCommonOutputDirectory Condition="'$(UseCommonOutputDirectory)' == ''">true</UseCommonOutputDirectory>
+    <CodeAnalysisRuleSet>$(RepoRoot)build\ruleset\Default.ruleset</CodeAnalysisRuleSet>
 
     <!-- Prevent vsix manifests from being copied to the shared output directory, makes build more deterministic -->
     <CopyVsixManifestToOutput>false</CopyVsixManifestToOutput>
```

Comment:
I think this is a setup only affecting property - but you can check by seeing if any extension.manifest get copied.

Reply:
Thats what I thought too, but I got build failures when I removed it from here. I haven't looked into it deeply, but it seemed like if this wasn't here, it didn't embed the manifest.

Reply:
Did setup projects get the same property?

Reply:
Just to follow up on this, yes this is in setup projects too. Once we work out (in a later task) why removing this fails the build, and merge in Tom's editorconfig branch, this whole `src\Directory.Build.props` file can be deleted.